src/components/layout/Layout.js

```diff
@@ -1,44 +1,40 @@
-import React, { useState } from "react";
-import SideMenu from "./SideMenu/SideMenu";
-import MenuContext from "../../store/menu-context";
-import {
-  StyledContent,
-  StyledDrawer,
-  StyledDrawerPaper,
-  StyledRoot,
-} from "./LayoutStyled";
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { StyledContent } from './LayoutStyled';
+import Menu from './Menu/Menu';
 
 const Layout = ({ children }) => {
-  const [menuCollapsed, setMenuCollapsed] = useState(false);
+	const [open, setOpen] = useState(false);
 
-  const menuToggleHandler = () => {
-    setMenuCollapsed((value) => !value);
-  };
+	const handleDrawerOpen = () => {
+		setOpen(true);
+	};
 
-  return (
-    <StyledRoot>
-      <MenuContext.Provider
-        value={{
-          isMenuCollapsed: menuCollapsed,
-          onMenuToggle: menuToggleHandler,
-        }}
-      >
-        <StyledDrawer
-          variant="permanent"
-          open={true}
-          menucollapsed={menuCollapsed ? 1 : 0}
-        >
-          <StyledDrawerPaper menucollapsed={menuCollapsed ? 1 : 0}>
-            <SideMenu
-              onMenuToggle={menuToggleHandler}
-              isMenuCollapsed={menuCollapsed}
-            />
-          </StyledDrawerPaper>
-        </StyledDrawer>
-        <StyledContent container>{children}</StyledContent>
-      </MenuContext.Provider>
-    </StyledRoot>
-  );
+	const handleDrawerClose = () => {
+		setOpen(false);
+	};
+
+	let location = useLocation();
+
+	useEffect(() => {
+		handleDrawerClose();
+	}, [location]);
+
+	return (
+		<Box sx={{ display: 'flex' }}>
+			<Menu
+				open={open}
+				handleDrawerClose={handleDrawerClose}
+				handleDrawerOpen={handleDrawerOpen}
+			/>
+			<StyledContent
+				container
+				onClick={handleDrawerClose}>
+				{children}
+			</StyledContent>
+		</Box>
+	);
 };

```

Comment:
This is how I see the main page layout: 
![Screenshot (51)](https://github.com/Greenstand/treetracker-wallet-admin-client/assets/19413810/d4c2982e-6224-4115-b8d1-bbf9828c3298)
Page content should fit the page and user should not see the vertical scroll.

Reply:
The same with a mobile view: 
![Screenshot (52)](https://github.com/Greenstand/treetracker-wallet-admin-client/assets/19413810/0da79845-44c8-4e2f-a096-3cf0e69ee450)

Reply:
I have removed those empty white spaces on my lastest git commit